### PR TITLE
Adding instructions for Debian systems.

### PR DESCRIPTION
**Pull request by jeremiah**
Library names differ across distros. *sigh*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
-# flat-manager
+ # flat-manager
```
Comment (mwleeds) [inline README.md]: Unnecessary whitespace change?

File: README.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
 flat-manager serves and maintains a Flatpak repository. You point it
 at an ostree repository and it will allow Flatpak clients to install
 apps from the repository over HTTP. Additionally, it has an HTTP API
 that lets you upload new builds and manage the repository.
 
-## Building the server
+## Building the server
```
Comment (mwleeds) [inline README.md]: Unnecessary whitespace change?

File: README.md (hunk 1 above)
Comment (jeremiah) [reply README.md]: Yes. :-)

File: README.md (hunk 2 above)
Comment (jeremiah) [reply README.md]: Indeed.

File: README.md (hunk 1 above)
Comment (jeremiah) [reply README.md]: I removed the white space change through squashing.

File: README.md (hunk 2 above)
Comment (jeremiah) [reply README.md]: I removed the white space change through squashing.

**Comment by mwleeds** [comment]
Looks like merge conflicts need to be resolved

**Comment by alexlarsson** [comment]
Yeah, i manually merged this by cherry picking the commits. Thanks!
